app/frontend/src/submissions/views/SubmissionsHome.vue

```diff
@@ -574,14 +576,121 @@ export default {
 
         // non-form fields that should be saved with form
         meta: {
-          drillerSameAsPersonResponsible: false
+          drillerSameAsPersonResponsible: false 
         }
       }
       this.componentUpdateTrigger = Date.now()
     },
+    groundwaterProtectionRegulationValidation(errors) {
+      const {
+        owner_full_name, 
+        owner_mailing_address,
+        owner_city,
+        owner_province_state,
+        owner_postal_code,
+      } = this.form
+
+      if (!owner_full_name) {
+        errors.owner_full_name = ['Owners Full Name Required.'];
+      }
+      if (!owner_mailing_address) {
+        errors.owner_mailing_address = ['Owners Mailing Address Required.'];
+      }
+      if (!owner_city) {
+        errors.owner_city = ['Owners City Required.'];
+      }
+      if (!owner_province_state) {
+        errors.owner_province_state = ['Owners Province or State Required.'];
+      }
+      if (!owner_postal_code) {
+        errors.owner_postal_code = ['Owners Postal Code Required.'];
+      }
+    },
+    validateWellIdentificationPlateFields(errors) {
+      const { 
+        well_class,
+        identification_plate_number,
+        well_identification_plate_attached,
+      } = this.form
+
+      let isWellIdentificationPlateToBeVerified = false;
+
+      if (well_class === WELL_CLASS.WATER_SUPPLY) { isWellIdentificationPlateToBeVerified = true; }
+      if (well_class === WELL_CLASS.INJECTION) { isWellIdentificationPlateToBeVerified = true; }
+      if (well_class === WELL_CLASS.RECHARGE) { isWellIdentificationPlateToBeVerified = true; }
```

Comment:
```suggestion
     const validatedWellClasses = [WELL_CLASS.WATER_SUPPLY, WELL_CLASS.INJECTION, WELL_CLASS.RECHARGE]
     const isWellIdentificationPlateToBeVerified = validatedWellClasses.includes(well_class);
```
something like this might be a little more DRY

Reply:
Much better, thanks.